### ISSUE TITLE
x86-32bit architecture support

### ISSUE THREAD

**Issue by ParkHanbum** [namhyung/uftrace#179]
I'm using ubuntu 16.04 LTS 32bit. meet compile error when using make. 

```
make[1]: *** /home/m/test/uftrace/arch/i386: No such file or directory.  Stop.
Makefile:183: recipe for target '/home/m/test/uftrace/arch/i386/uftrace.o' failed
make: *** [/home/m/test/uftrace/arch/i386/uftrace.o] Error 2

m@ubuntu:~/test/uftrace$ uname -a
Linux ubuntu 4.10.0-28-generic #32~16.04.2-Ubuntu SMP 
Thu Jul 20 10:19:13 UTC 2017 i686 i686 i686 GNU/Linux
```

**Comment by namhyung** [comment]
Hello, unfortunately i386 arch is not supported for now.

**Comment by ParkHanbum** [comment]
is someone working on it? if not, may I try it?

**Comment by namhyung** [comment]
It would be great if you could work on it!

**Comment by ParkHanbum** [comment]
@namhyung thanks you! I'll try my best~~!

**Comment by namhyung** [comment]
Merged #282.
